index.html

```diff
@@ -1,92 +1,131 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Microverse project</title>
-    <link rel="stylesheet" href="./style.css">
-</head>
-<body>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
     <header id="header">
-        <h1 id="title">freeCodeCamp Survey Form</h1>
-        <p id="description"> <i>Thank you for taking the time to help us improve the platform</i></p>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">
+        <i>Thank you for taking the time to help us improve the platform</i>
+      </p>
     </header>
     <div id="container">
-        <form action="" id="survey-form">
-            <div class="form-control">
-                <label for="" id="name-label">Name</label><input required type="text" id="name" placeholder="Enter your name please">
-            </div>
-            <div class="form-control">
-                <label for="" id="email-label">Email</label><input required id="email" type="email" placeholder="Enter your Email Adress">
-            </div>
-            <div class="form-control">
-                <label for="" id="number-label">Age</label><input min="3" max="8" size="5" required id="number" type="number" placeholder="Enter your number">
-
-            </div>
-            <div class="form-control">
-                <label for="">Which option best describes your current role?</label><input required type="text">
-            </div>
-            <div class="form-control">
-                <p>What is your favorite feature of freeCodeCamp?</p>
-                <select name="" id="dropdown">
-                    <option value="">All</option>
-                    <option value="challeges">Chellenges</option>
-                    <option value="">course</option>
-                    <option value="">Full time job</option>
-                    <option value="">Other</option>
-                </select>
-            </div>
-            <div class="form-radio">
-                <p>What would you like to see improved? (Check all that apply)</p>
-                <input type="radio" id="male" name="gender" value="male">
-                <label for="male">Male</label><br>
-                <input type="radio" id="female" name="gender" value="female">
-                <label for="female">Female</label><br>
-                <input type="radio" id="other" name="gender" value="other">
-                <label for="other">Other</label>
-            </div>
-            <div class="form-control">
-                <select name="" id="dropdown">
-                    <option value="">All</option>
-                    <option value="challeges">Chellenges</option>
-                    <option value="">course</option>
-                    <option value="">Full time job</option>
-                    <option value=""> <span>Other</span> </option>
-                </select>
-
-            </div>
-            <div class="form-control">
```

Comment:
- [ ] Semantic tags more accurately describe the content that is on the page. Describing the contents of the page more semantically can have many benefits. it would be better to use them here
- using ul and li tags only makes your code more readable and structured.